test/invoke.test.ts

```diff
@@ -255,6 +255,107 @@ describe('invoke', () => {
     service.send('FORWARD_DEC');
   });
 
+  it.only('should forward events to services if autoForward: true before processing them', done => {
```

Comment:
```suggestion
  it('should forward events to services if autoForward: true before processing them', done => {
```

Reply:
hah, this was leftover as I've started to work on this one before you've merged in jest support and forgot to remove this when i've applied my stashed changed

fixed